recipe/meta.yaml

```diff
@@ -19,14 +19,14 @@ build:
     - lib/R/lib/
     - lib/
 
-requirements:  # [not win32]
-  build:  # [not win32]
-    - r-base  # [not win32]
-    - posix  # [win64]
-    - m2w64-toolchain  # [win64]
-    - gcc  # [not win]
```

Comment:
> You need to keep all the general win selectors of course. Only it is not necessary any more to completely hide the requirements section for win32, because the build->skip is now handled correctly.

But that is what I want to fix :smile: keeping the various selectors is a hack and we should not rely on that.